src/react-apps/applications/shared/src/theme/altinnStudioTheme.tsx

```diff
@@ -17,36 +43,40 @@ const theme = {
       },
     },
     MuiTypography: {
-      h4: {
+      h1: {
         fontSize: 36,
-      },
-      h5: {
-        fontSize: 24,
-      },
-      h6: {
-        fontSize: 18,
-      },
-      subtitle2: {
-        fontSize: 16,
-      },
-      body1: {
-        fontSize: 14,
-      },
-      body2: {
-        fontSize: 14,
-        fontWeight: 500,
       }
     }
   },
+  altinnPalette: {
+    //These are the primary colors used in altinn: https://altinn.github.io/designsystem-styleguide/retningslinjer-altinn/farger.html
+    primary: {
+      blueDarker: '#022F51',
+      blueDark: '#0062BA',
+      blue: '#1EAEF7',
+      blueLight: '#CFF0FF',
+      green: '#17C96B',
+      greenLight: '#D4F9E4',
+      red: '#E23B53',
+      redLight: '#F9CAD3',
+      purple: '#3F3161',
+      purpleLight: '#E0DAF7',
+      yellow: '#FFDA06',
+      yellowLight: '#FBF6BD',
+      black: '#000',
+      grey: '#6a6a6a',
+      greyMedium: '#BCC7CC',
+      greyLight: '#EFEFEF',
+      white: '#FFF',
+    }
+  },
   palette: {
     primary: {
-      main: '#0062BA',
-      dark: '#022F51',
-      light: '#CFF0FF',
+      main: '',
     },
+    // Colors that are not part of the altinn color palette but is still used
     secondary: {
-      light: '#efefef',
-      main: '#FFFFFF',
+      main: '',
```

Comment:
Is this supposed to be empty?

Reply:
main has to be there or else it will show error